core/nuxeo-operation.js

```diff
@@ -75,6 +75,16 @@ import './nuxeo-page-provider.js';
           notify: true,
         },
 
+        /** Indicates if the request behind this operation cannot be cancelable.
```

Comment:
Either
```suggestion
        /** Indicates if the request behind this operation cannot be canceled.
```
Or
```suggestion
        /** Indicates if the request behind this operation is not cancelable.
```

Reply:
:

Reply:
Go for the first proposition:  /** Indicates if the request behind this operation c**annot be canceled**.